target/linux/layerscape/patches-5.4/801-audio-0039-MLK-16224-6-ASoC-fsl_sai-fix-DSD-suspend-resume.patch

```diff
@@ -36,7 +36,7 @@ Reviewed-by: Shengjiu Wang <shengjiu.wang@nxp.com>
 +	return 0;
 +}
 +
- static struct snd_soc_dai_driver fsl_sai_dai = {
```

Comment:
Change from "fsl_sai_dai" to "fsl_sai_dai_template" is not part of the original patch, it was done later in a different context. Now it depends on the order the patches are applied - the kernel should be compilable with no errors after each patch is applied, one by one.

Reply:
@ViorelSuman - I do not find a reference to "fsl_sai_dai" in the PR.... just:
* fsl_sai_dai_probe(struct snd_
* fsl_sai_dai_resume(struct snd_soc_dai *cpu_dai)
* fsl_sai_dai_template

Reply:
Right, ok, so you apply those patches on top of the upstream version where "fsl_sai_dai_template" is present I think.
Then it's fine with me, your patch is ok.

Reply:
Thank you for the time to review